src/pages/Landing/About.tsx

```diff
@@ -0,0 +1,179 @@
+    <VStack
+      maxW="500px"
+      marginLeft={{ base: 'auto', md: flip ? 'auto' : 0 }}
+      marginRight={{ base: 'auto', md: flip ? 0 : 'auto' }}
+      align="left"
+      spacing={2}
+      gridColumn={{ base: 1, md: flip ? 2 : 1 }}
+    >
+      <Text
+        color="orangeText"
+        fontSize="md"
+        fontWeight="500"
+        textTransform="uppercase"
+      >
+        {hint}
+      </Text>
+      <Text fontWeight="500" fontSize="2xl" textTransform="uppercase">
+        {title}
+      </Text>
+      <Text color="gray.500" fontSize="md">
+        {description}
+      </Text>
+    </VStack>
+    <Image
+      w="375px"
+      h="275px"
+      marginLeft={{ base: 'auto', md: flip ? 0 : 'auto' }}
+      marginRight={{ base: 'auto', md: flip ? 'auto' : 0 }}
+      marginTop={{ base: '40px', md: 0 }}
+      gridColumn={{ base: 1, md: flip ? 1 : 2 }}
+      gridRow={{ base: 2, md: 1 }}
+      bg="gray.300"
+    />
+  </SimpleGrid>
+);
+
+const LandingAbout = () => (
+  <Box>
+    <Container maxW="6xl" centerContent py={20}>
+      <VStack spacing={12}>
+        <VStack spacing={4}>
+          <Text fontSize="4xl">So... What is this platform all about?</Text>
+          <Text fontSize="lg" color="gray.500">
+            TL;DR Well... it’s basically like Pinterest, but for students
+            looking to do digital project.
+          </Text>
+        </VStack>
+        <SimpleGrid columns={{ base: 1, sm: 2, lg: 3 }}>
+          <AboutIconBox
+            icon={HeartIcon}
+            color="purple"
+            title="Create posts"
+            description="We have great user interface that makes navigating accountiing tasks fun and easy."
+          />
+          <AboutIconBox
+            icon={HeartIcon}
+            color="orange"
+            title="Recruit team members"
+            description="We have great user interface that makes navigating accountiing tasks fun and easy."
+          />
+          <AboutIconBox
+            icon={HeartIcon}
+            color="green"
+            title="Collaborators"
+            description="We have great user interface that makes navigating accountiing tasks fun and easy."
+          />
+          <AboutIconBox
+            icon={HeartIcon}
+            color="pink"
+            title="Categories"
+            description="We have great user interface that makes navigating accountiing tasks fun and easy."
+          />
+          <AboutIconBox
+            icon={HeartIcon}
+            color="yellow"
+            title="Chat feature"
+            description="We have great user interface that makes navigating accountiing tasks fun and easy."
+          />
+          <AboutIconBox
+            icon={HeartIcon}
+            color="blue"
+            title="100% free"
+            description="We have great user interface that makes navigating accountiing tasks fun and easy."
+          />
+        </SimpleGrid>
+        <AboutImageBox
```

Comment:
should we split iconbox & imagebox into separate files, and then import them in various parts of the "landing page"? for example if in the future we want to take out imagebox from the "about" part of the landing page and use it in a different part?

Reply:
I would suggest keeping it the current way until we get to a point where we reuse this component. We can refactor this out later if we want to. Currently it's built very specifically for the homepage.